bot/sx-bot.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/bash
+
+DESTINATION_BRANCH=gh-pages
+
+function notify-done {
+    local title
+    local message
+    title="SX Tag Bot"
+    message="Finished retrieving tag lists"
+    case $(uname | tr '[[:upper:]]' '[[:lower:]]') in
+        darwin)
+            terminal-notifier \
+                -message ${message} \
+                -title ${title} \
+                -sound default
+            ;;
+        *)
+            echo ${message}
+    esac
+}
+
+function generate-tags {
+    emacs -Q --batch \
+          -L "./" -L "./bot/" -l sx-bot \
+          -f sx-bot-fetch-and-write-tags
+    ret = $?
+    notify-done
+    return ${ret}
+}
+
+git branch ${DESTINATION_BRANCH} &&
+    git pull &&
+    generate-tags &&
+    git stage data/ &&
+    git commit -m "Update tag data" &&
```

Comment:
The commit date is included in the commit data – do you think it's necessary? I'm not sure of the possibilities with other tools interfacing with the repository, so I'm just going off my personal experience.